source/grid/grid_reordering.cc

```diff
@@ -532,101 +641,87 @@ namespace internal
       // this
       //
       // note that in 2d, each of the Deltas can at most
-      // contain two elements. we indicate non-used elements
-      // of these sets by invalid unsigned ints; if the set has
-      // only one element, then we use the first
-      unsigned int Delta_k[2] = { numbers::invalid_unsigned_int,
-                                  numbers::invalid_unsigned_int
-                                };
-      unsigned int Delta_k_minus_1[2] = { cells[cell].edge_indices[local_edge],
-                                          numbers::invalid_unsigned_int
-                                        };
-      while (Delta_k_minus_1[0] != numbers::invalid_unsigned_int)   // while set is not empty
+      // contain two elements, whereas in 3d it can be arbitrarily many
+      EdgeDeltaSet<dim> Delta_k;
+      EdgeDeltaSet<dim> Delta_k_minus_1;
+      Delta_k_minus_1.insert (cells[cell].edge_indices[local_edge]);
```

Comment:
It's a valid point, but I value the equivalence with the paper more. I also already use the (lowercase) name `delta` in this code.